.babelrc

```diff
@@ -1,4 +0,0 @@
-{
```

Comment:
Remove the empty README files

Reply:
Also if we dont need a circleci job for this then delete the config file. Otherwise pushes will still trigger circleci jobs that will fail